src/main/java/com/palantir/gradle/versions/GetVersionPlugin.java

```diff
@@ -36,43 +36,44 @@
 
     @Override
     public void apply(Project project) {
-        project.getExtensions().getExtraProperties().set("getVersion", new Closure<String>(project, project) {
```

Comment:
I guess we need the "all on one line lambda" thing to work with anonymous classes too?

Reply:
yep! will collate all the nasty cases here and start addressing them once I fix the remaining two integration tests.